.github/workflows/nessie.yaml

```diff
@@ -145,7 +145,7 @@ jobs:
           NESSIE_STORAGE_NAMESPACE: s3://nessie-system-testing/${{ github.run_number }}-${{ matrix.cataloger }}
           NESSIE_AWS_ACCESS_KEY_ID: ${{ secrets.NESSIE_AWS_ACCESS_KEY_ID }}
           NESSIE_AWS_SECRET_ACCESS_KEY: ${{ secrets.NESSIE_AWS_SECRET_ACCESS_KEY }}
-        run: go test -v ./nessie --system-tests
+        run: go test -v --coverprofile=nessie-s3-cover.out --cover ./nessie --system-tests
```

Comment:
I am really confused about this.
You run Nessie with coverprofile, but you're really interested in the coverage of the lakeFS app running inside a container. What would covering Nessie's code give you?

Reply:
Yikes, true!